src/tasks/daily.ts

```diff
@@ -349,6 +396,15 @@ export function configureSnojo(): void {
 }
 
 export const DailyTasks: Task[] = [
+  {
+    name: "Chibi Buddy",
+    ready: () =>
+      !get("_chibiChanged", true) &&
+      (have($item`ChibiBuddy™ (on)`) || have($item`ChibiBuddy™ (off)`)),
```

Comment:
```suggestion
    ready: () =>have($item`ChibiBuddy™ (on)`) || have($item`ChibiBuddy™ (off)`),
```
Don't need to double up on the completion criteria